model/src/main/proto/oppia_logger.proto

```diff
@@ -9,39 +9,94 @@ option java_multiple_files = true;
 message EventLog {
   // The timestamp of the event.
   int64 timestamp = 1;
-  // The action happening in the event.
-  EventAction action_name = 2;
   // The priority of the event.
-  Priority priority = 3;
+  Priority priority = 2;
   // The activity context of the event.
-  Context context = 4;
+  Context context = 3;
```

Comment:
Lets not delete proto fields like this, instead you can reserve field number 2 and keep priority and context fields asi s 
i.e. 
```
int64 timestamp = 1;
reserved 2;
Priority priority = 3;
Context context = 4;
```
 (or you can keep the reserved keyword at the top of the message before all the other params)

See https://developers.google.com/protocol-buffers/docs/proto3#updating for details about how to update a proto message!

Reply:
Insightful! done -- thanks!